db/seeds.rb

```diff
@@ -5,3 +5,27 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+Recipe.destroy_all
+Ingredient.destroy_all
+RecipeIngredient.destroy_all
+
+
+# recipe 1 should use 1,2,3,4 and recipe 2 should use 1,2,5
+@recipe_1 = Recipe.create!(name: "Spaghetti", complexity: 3, genre: "Italian")
+@recipe_2 = Recipe.create!(name: "Burger", complexity: 1, genre: "American")
+
+@ingredient_1 = Ingredient.create!(name: "Ground Beef", cost: 2)
+@ingredient_2 = Ingredient.create!(name: "Salt", cost: 4)
+@ingredient_3 = Ingredient.create!(name: "Noodles", cost: 1)
+@ingredient_4 = Ingredient.create!(name: "Tomatoes", cost: 2)
+@ingredient_5 = Ingredient.create!(name: "Burg Bun", cost: 1)
+
+@spaghetti_1 = RecipeIngredient.create!(recipe_id: "#{@recipe_1.id}", ingredient_id: "#{@ingredient_1.id}")
```

Comment:
These look good. I don't think you need to string interpolate them though. You should just be able to
```ruby
@spaghetti_1 = RecipeIngredient.create!(recipe: recipe_1, ingredient: ingredient_1)
```